kore/src/Kore/Repl/Data.hs

```diff
@@ -825,3 +856,118 @@ substituteAlias
     toString = \case
         SimpleArgument str -> str
         QuotedArgument str -> "\"" <> str <> "\""
+
+createOnePathClaim
+    :: Claim claim
+    => (claim, CommonStrategyPattern)
+    -> Rule.OnePathRule Variable
+createOnePathClaim (cl, cpatt) =
+    Rule.OnePathRule
+    $ Rule.RulePattern
+        { left = pattToLeftPatt cpatt
+        , right = claimToRightPatt cl
+        , requires = Predicate.makeTruePredicate
+        , ensures = claimToEnsures cl
+        , attributes = Default.def
+        }
+  where
+    pattToLeftPatt
+        :: CommonStrategyPattern
+        -> TermLike Variable
+    pattToLeftPatt =
+        toTermLike . fromJust . extractUnproven
+    claimToRightPatt
```

Comment:
`claimToRightPatt` and `claimToEnsures` are simple enough that they might be better inline, e.g. `right = Rule.right . coerce $ pattern`